scripts/reaction.js

```diff
@@ -0,0 +1,48 @@
+// Description
+//   Allows users to store and bring out reaction images/gifs
+// 
+// Commands:
+//   !mrw <reaction_string>
+// 	 uqcsbot new reaction <reaction_string> <url to reaction image>
+// 
+
+module.exports = function (robot) {
+  robot.hear(/^!mrw (.+)/i, function (res) {
+	  var rStr = res.match[1];
+	  if(!rStr) return;
+	  robot.showReaction(rStr, res)
+  });
+  
+  robot.hear(/^uqcsbot new reaction (\w+\ [^ \n]+)/i, function (res) {
```

Comment:
you can get rid of the ^uqcsbot part and just use robot.respond instead. This functions a lot better, because currently you can't PM uqcsbot things that start with ^ in the regex. Can you change that pls?